src/backend/base/langflow/graph/graph/base.py

```diff
@@ -651,12 +653,13 @@ def set_run_id(self, run_id: uuid.UUID | None = None) -> None:
 
         self._run_id = str(run_id)
 
-    async def initialize_run(self) -> None:
+    async def initialize_run(self, session: AsyncSession) -> None:
```

Comment:
We should make this optional or even use the `session_scope` in the `start_tracers` method. I'm not sure adding the session to the Graph is a good idea.